experimental/agents/README.md

```diff
@@ -0,0 +1,84 @@
+# AWS Controllers for Kubernetes (ACK) Agent
+
+A powerful Strands agent that uses Claude 3.7 Sonnet on AWS Bedrock with tools for working with AWS Controllers for Kubernetes (ACK).
+
+## Overview
+
+The ACK Agent provides a conversational interface to help you work with AWS Controllers for Kubernetes:
+
+- Clone and manage ACK repositories (code-generator, runtime, service controllers)
```

Comment:
My two cents here is that an agent shouldn't be responsible of cloning and setting up the environment. This strikes me as a determinstic task that needs to be handle by tooling instead, for example a prow job would prepare the repos and then call the agent to do it's magic. cc @knottnt @michaelhtm